jupyter_scheduler/models.py

```diff
@@ -92,16 +85,29 @@ class CreateJob(BaseModel):
     compute_type: Optional[str] = None
 
 
+class Output(BaseModel):
+    """Output model for job outputs
+    in Jupyter Lab root, this is
+    what the front end uses to render
+    links to outputs
+    """
+
+    display_name: str
+    output_format: str
+    output_path: Optional[str] = None
+
+
 class DescribeJob(CreateJob):
     job_id: str
-    output_uri: str
+    outputs: List[Output] = []
     url: str
     create_time: int
     update_time: int
     start_time: Optional[int] = None
     end_time: Optional[int] = None
     status: Status = Status.CREATED
     status_message: Optional[str] = None
+    downloaded: Optional[bool] = False
```

Comment:
We we computing this in the list/describe handler, or does each backend need to compute this based on the outputs?

Reply:
Each backend will compute this in their list/describe handlers, but we are providing a helper in the `BaseScheduler` class that computes the `outputs`.